.travis.yml

```diff
@@ -28,7 +28,7 @@ env:
     #- TARGET_PLATFORM=ubuntu/xenial
     - TARGET_PLATFORM=ubuntu/focal
```

Comment:
It appears that all distros are commented out in here. Please fix this, and do another push